src/components/toggle/ThemeToggle.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+
+import './Toggle.scss';
+import Toggle from 'react-toggle';
+
+class ThemeToggle extends Component {
+  constructor(props) {
+    super(props);
+
+    if(JSON.parse(localStorage.getItem('dark-mode')) === true) {
+      document.body.classList.add('dark-mode');
+    }
+
+    this.state = {
+      darkMode: JSON.parse(localStorage.getItem('dark-mode'))
+    }
+
+    this.handleModeChange = this.handleModeChange.bind(this);
+  }
+
+  getState() {
+    return this.state.darkMode;
+  }
+
+  handleModeChange() {
+    if(!this.state.darkMode) {
+      document.body.classList.add('dark-mode');
+    } else {
+      document.body.classList.remove('dark-mode');
+    }
+
+    this.setState({
+      darkMode: (!this.state.darkMode)
+    });
+    localStorage.setItem('dark-mode', !this.state.darkMode);
+  }
```

Comment:
Great! This pattern is super simple and seems to work well nice job 🎉 

Wondering if we should scope the key further here something like `neo3-preview-dark-mode` maybe this doesnt matter

Reply:
I can tweak it no problem :)